packages/shared/src/components/BlockInjectButton/BlockInjectButton.tsx

```diff
@@ -12,6 +13,17 @@ import React from 'react';
 import { DragEventHandler, MouseEventHandler, useRef, useState } from 'react';
 import { joinClassNames } from '../../utilities/react/joinClassNames';
 import { BlockInjectButtonProps } from './types';
+import { FileExtension, FileExtensionSets } from '@frontify/app-bridge';
+
+const horizontalGroupStyles =
+    '[&:not(:first-child)]:tw-border-l-0  first:tw-rounded-tl first:tw-rounded-bl last:tw-rounded-tr last:tw-rounded-br';
+const verticalGroupStyles =
+    '[&:not(:first-child)]:tw-border-t-0  first:tw-rounded-tl first:tw-rounded-tr last:tw-rounded-br last:tw-rounded-bl';
```

Comment:
```suggestion
const horizontalGroupStyles = '[&:not(:first-child)]:tw-border-l-0  first:tw-rounded-bl last:tw-rounded-tr';
const verticalGroupStyles = '[&:not(:first-child)]:tw-border-t-0  first:tw-rounded-tr last:tw-rounded-bl';
```

removed duplicated classes added within another suggestion directly as they are used anyways

Add these two "removed" classes `first:tw-rounded-tl last:tw-rounded-br` directly to the button on line 87

Reply:
changed in [fcc9555](https://github.com/Frontify/guideline-blocks/pull/539/commits/fcc955505aa583ab7cccc1ebaa775121e52ea118). Did you mean it like this?

Reply:
no, these 2 classes: `first:tw-rounded-tl last:tw-rounded-br` are used as verticalStyle and horizontalStyle so they are not related to vertical or horizontal so they are always on the button and the rest is conditional, hit me up if there is further unclarity

Reply:
Fair enough. I thought it's more readable if we keep all rounded tailwind classes within one condition, but you're right. Changed it